### ISSUE TITLE
Section 2.3: image creation info

### ISSUE THREAD

**Issue by bamford** [willettk/gzhubble#104]
Deciding on the image creation method and parameters was a bit of a battle.  I've been hunting through long email threads and it is hard to know the details of exactly what code Roger used.  When I looked into this previously I came up with:

> All HST surveys except COSMOS ...
> scale = 35, nonlinearity = 2
> But that scale probably has a factor of 1.0e-4, and might vary slightly by band.  I can't find the code itself.
> 
> For COSMOS...
> scale = 3  \* 1.0d-4 \* [0.8, 1.0, 1.0]
> nonlinearity = 3

I'm also fairly sure that Roger scaled the original count images by their exposure time, but I don't know exactly how.

All this means that it probably doesn't help to give numbers for the intensity scaling in this paper.  Just say that the scalings were adjusted in an attempt to generally match the appearance of the SDSS images in GZ1 and 2.

I don't think it is worth giving eqn. 1 either, especially without defining r.

Here's my rewrite of sec 2.3 (merging in @vrooje's suggestions in #98 - please check!):

``` LateX
The images used for classification in GZH were color-composite JPEGs made from
multi-band data. These were created followed the method of \citet{lup04}, which
preserves colour information irrespective of intensity.  An asinh intensity
mapping was applied to enhance the appearance of faint features while avoiding
saturating galaxy centres.  When choosing the relative scalings of the filter bands,
we tried to reproduce a similar color appearance to the SDSS images in previous
iterations of Galaxy Zoo.

Many of the Legacy surveys described in Section~\ref{ssec:legacy_surveys}
provided images in only two filters. For these, the shorter-wavelength band was mapped
to the blue channel, the longer-wavelength band to the red channel, and the green
channel created by taking the arithmetic mean of the two. The bands used in each of
the surveys are listed in Table~\ref{tbl:gzh_numbers}.  Although four bands were available
for the GOODS survey, only two bands were used to create the original 2-epoch images,
for consistency with AEGIS and GEMS. The 2-epoch GOODS-N and
GOODS-S images were created using different filters --- this was a deliberate choice made
so that the GEMS images could be directly compared with the overlapping coverage
of GOODS-S (Figure~\ref{fig:filtercurves}).

Only 2-epoch GOODS images were included at the launch of GZH.  Deeper, 5-epoch, GOODS
images were added into GZH later, in March 2015. For these images we made use of the full
four-band data by using the arithmetic mean of \Bband{} and \Vband{} in the blue channel,
\Iband{} in the green channel, and \zband{} in the red channel.

The COSMOS survey only provided \Iband{} HST imaging at the time of the GZH launch.
For these galaxies, we used ``pseudocolor'' images created by using the ACS \Iband{} data
as an illumination map and ground-based imaging from the \subaru{} telescope in
$B_J$, $r^+$, and $i^+$ filters to provide color information
\citep[see][for further details]{gri12}. This resulted in images with the angular
resolution of \hst{} ($\sim0.05$~\arcsec/pixel) for the overall intensity, but color gradients
at ground-based resolution, with seeing between 0.95\arcsec{} and 1.05\arcsec{} \citep{tan07}.

Stripe~82 single-epoch images were taken directly from the DR7 SDSS Skyserver,
which combined $g^{\prime}$, $r^{\prime}$, and $i^{\prime}$ exposures into the
RGB channels. The co-added Stripe~82 images were assembled from runs 106 and
206 in DR7 and processed into color composites in a corresponding manner.

In some cases we found that attempting to emphasize faint features in the images resulted
in the sky noise taking the appearance of brightly coloured speckles. This impaired the
aesthetics of the images and might have been a distraction to visual classification.
To counteract this, a soft-edged object mask was applied to the color images and a
desaturation operation performed. This masked procedure was effective in preserving
the color balance for galaxies, retaining the visibility of faint features, but reducing the
color contrast in the sky noise, greatly improving the appearance of the images.
This solution was applied to the coadded Stripe~82, COSMOS and 5-epoch GOODS images.
```

**Comment by bamford** [comment]
OK, now see that a bigger reorganisation was proposed in #95, but hopefully this provides some useful / more correct text for that.

**Comment by vrooje** [comment]
I think this is completely addressed by #115 - do reopen if I missed something though.
